modules/core/src/main/java/org/apache/ignite/client/ClientChannelDisconnectListener.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client;
+
+import javax.cache.configuration.CacheEntryListenerConfiguration;
+import org.apache.ignite.cache.query.ContinuousQuery;
+
+/**
+ * This interface can be implemented by {@link ContinuousQuery} local listeners or by cache entry listeners
+ * registered via {@link ClientCache#registerCacheEntryListener(CacheEntryListenerConfiguration)} to get notification
+ * about client channel disconnected event.
+ */
+public interface ClientChannelDisconnectListener {
```

Comment:
`ClientDisconnectListener` seems to be a better name - `channel` sounds like an implementation detail.

Reply:
Ok